index.html

```diff
@@ -0,0 +1,107 @@
+      </ul>
+    </nav>     
+  </header>
+          <div class="form">
+        <h2>Handcrafted, home-mad masterpieces</h2>
+        <form id="form" action="https://wwww.freecodecamp.com/email-submit">
+          <input
+                 type="email"
+                 placeholder="Enter your email address"
+                 name="email"
+                 id="email"
+                 required
+              />
+          <input id="submit" type="submit" value="Get Started"/>
+        </form>
+      </div>
+    <div id="features"> 
+    <div class="card">
+      <div class="icon"><i class="fa fa-3x fa-fire"></i></div>
+        <div class="content">
+          <h2> Premium Materials</h2>
+            <p>
+              Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.
+            </p>
+        </div>     
+      </div>
+      
+      <div class="card">
+        <div class="icon"><i class="fa fa-3x fa-truck"></i></div>
+        <div class="content">
+          <h2>Fast Shipping</h2>
+          <p>
+            We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.
+          </p>
+        </div>        
+      </div>
+      
+      <div class="card">
+        <div class="icon"><i class="fa fa-3x fa-battery-full"></i></div>
+        <div class="content">
+          <h2>Quality Assurance</h2>
+          <p>
+            For every purchase you make, we will ensure there are  no damages or faults and we will check and test the pitch of your instrument.
+          </p>
+        </div>
+      </div>   
+    </div>
+    
+    <div id="how-it-works">
+      <iframe 
+      width="560" 
+      height="315" 
+      src="https://www.youtube.com/embed/f9ip1iSAptQ" 
+      frameborder="0" 
+      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+      allowfullscreen
+      ></iframe>
+    </div>
+    <div id="pricing">
+      <div class="card">
+        <h2>TENOR TROMBONE</H2>
+        <strong>$600</strong>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum dolor</p>
+        <button>Select</button>
+      </div>
+        <div class="card">
+        <h2>BASS TROMBONE</H2>
+        <strong>$900</strong>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum dolor</p>
+        <button>Select</button>
+      </div>
+           <div class="card">
+        <h2>VALVE TROMBONE</H2>
+        <strong>$1200</strong>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum dolor</p>
+        <button>Select</button>
+      </div>
+    </div>
+    
+    <footer>
+      <ul>
+        <li><a href="#">Privacy</a></li>
+        <li><a href="#">Terms</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>  
+      <p>Copyright 2016, Original Trombones</p>
+    </footer>
```

Comment:
Use Flex Box in this part too, yeah again... but remember that practice makes perfect... :)